content/source/docs/enterprise/private/automating-the-installer.html.md

```diff
@@ -146,7 +145,11 @@ The following apply to every installation:
 
 #### The following apply if you've chosen `external` for `production_type`:
 
-- `postgres_url` — (Required) the database URL
+- `pg_user` — (Required) PostgreSQL user name to connect as
```

Comment:
I might omit "name" here since we just use user in the next line, but I don't feel strongly.